utils/update_json_files.py

```diff
@@ -33,6 +33,25 @@ def clean_category(category_raw):
     if "## " in category_raw:
         return category_raw[3:]
 
+def validate_bookcover(book_details):
+    """
+    Check if goodreads returns a nophoto
+    Use open library to fetch the book cover
+    based on ISBN
+
+    Args:
+        book_details: Book info returned as json by goodreads API
+
+    Returns:
+        This API checks for book cover, and returns with a valid
+        bookcover if nophoto found on goodreads, using openlibrary
+    """
+    no_photo_url='https://s.gr-assets.com/assets/nophoto/book/111x148-bcc042a9c91a29c1d680899eff700a03.png'
```

Comment:
I think we can make this a bit more generalized and check for `https://s.gr-assets.com/assets/nophoto/book/` instead? Maybe there are some cases in which the ending part is different?